dojson/contrib/marc21/fields/bd1xx.py

```diff
@@ -312,5 +312,5 @@ def main_entry_uniform_title(self, key, value):
         'field_link_and_sequence_number': utils.force_list(
             value.get('8')
         ),
-        'nonfiling_characters': key[3] if key[3] else '_',
+        'nonfiling_characters': utils.int_with_default(key[3], '_'),
```

Comment:
LOC is using ` ` (empty space) as a default option, someone else (I don't remember by now, by @tiborsimko showed me another specification) is actually using `#` as default and we decided to go for `_` (as it was similar to the convention used on CDS). With `None` - can you easily say if the MARCXML contained `ind2` that was empty or if it didn't contained the `ind2` at all ?
Also, is it really worth changing it (for me it's just a matter of convention we used) ? It will only add more work to change all the mappings again (yet).